package.json

```diff
@@ -131,16 +131,15 @@
     "not IE_Mob 11"
   ],
   "engines": {
-    "node": ">=11.0",
+    "node": ">=14.0",
     "yarn": "^1.7.0"
   },
   "dependencies": {
     "@babel/runtime": "^7.12.5",
-    "@babel/runtime-corejs3": "^7.17.0",
     "@peculiar/webcrypto": "^1.4.0",
     "Base64": "1.1.0",
     "atob": "^2.1.2",
-    "broadcast-channel": "^4.13.0",
+    "broadcast-channel": "^4.14.0",
```

Comment:
```
"broadcast-channel": "~4.14.0",
```
I think we should use `~` for `broadcast-channel`, so we only automatically pull in patch versions

Reply:
Done